libs/pages/services/src/lib/feature/deployments/deployments-feature.tsx

```diff
@@ -70,7 +68,7 @@ export function Deployments() {
   }, [dispatch, environmentId, projectId, environment])
 
   return (
-    <DeploymentsPage
+    <Deployments
```

Comment:
And `DeploymentsPage` becomes `Deployments`

Reply:
same here